AnimateQuotes.js

```diff
@@ -82,7 +100,7 @@ export default function AnimateQuotes(props) {
       height: '100%',
       width: '100%',
     }}>
-      {props.children}
+      <Quotes quote={{ quote, setQuote }} />
```

Comment:
Quotes component nested within AnimateQuotes parent from within component return statement as opposed to previously being wrapped within the parent at the App component's return statement which was utilizing props.children.